drivers/tx_client_reports/app/models/tx_client_reports/attachment_three_report.rb

```diff
@@ -13,6 +13,146 @@ def initialize(filter)
       @filter = filter
     end
 
+    def attachment_three_headers
+      [
+        'Row Number',
+        'Warehouse ID',
+        # 'Project Name',
+        'Date of First Service',
+        # 'Report Start Date',
+        # 'Entry Started after Report Start',
+        'Fort Worth Resident?',
+        'Service Location',
+        'Name',
+        'Street Address',
+        'Age',
+        'Male',
+        'Female',
+        'Hispanic/Latin(a)(o)(x)',
+        'Non-Hispanic/Non-Latin(a)(o)(x)',
+        'American Indian, Alaska Native, or Indigenous',
+        'Asian or Asian American',
+        'Black, African American, or African',
+        'Native Hawaiian or Pacific Islander',
+        'White',
+        'American Indian, Alaska Native, or Indigenous and White',
+        'Black, African American, or African and White',
+        'American Indian, Alaska Native, or Indigenous and White and Black, African American, or African',
+        'Other Multiple Race Combinations',
+        'Disabled (Yes)',
+        'Disabled (No)',
+        'hhsize',
+        'HoH Monthly Income',
+        'FHOH (Yes)',
+        'FHOH (No)',
+      ]
+    end
+
+    def attachment_three_rows
+      rows.map.with_index do |row, index|
+        [
+          index + 1,
+          row[:client_id],
+          # row[:project_name],
+          row[:service_date],
+          # row[:report_start],
+          # ('X' if row[:entry_after_start]),
+          row[:fort_worth_resident],
+          row[:service_location],
+          row[:client_name],
+          row[:street_address],
+          row[:age],
+          ('1' if row[:genders].include?(0)),
```

Comment:
I am curious as to why the view put an 'X' for boolean values, in this one you put a '1', and on the household report it is 'Yes'

Reply:
So, in your first pass, you had used `X`.  I just based the current versions off the new spec.  One tab expects "Yes" or "No" the other "1" or blank.  I assume because they are different entities that need the sheets and haven't coordinated.

Reply:
I'm pretty sure the X's came from the same thinking ("This is what is in the sample file, I'll do the same")